server/routes/api.ts

```diff
@@ -339,4 +340,12 @@ router.get(
 router.post('/reminders/cancel', cancelReminderHandler);
 router.post('/reminders/reactivate', reactivateReminderHandler);
 
+if (featureSwitches.cspSecurityAudit) {
+	router.post('/csp-audit-report-endpoint', (req, res) => {
+		const parsedBody = JSON.parse(req.body.toString());
+		log.warn(JSON.stringify(parsedBody));
```

Comment:
looking at the screenshot in the description, it ends up as double nested JSON.  Is there a way to inline this JSON? or at least prettyPrint i.e. 2 space indent etc?
No worries if not but it might help!